client/core.js

```diff
@@ -176,12 +176,11 @@
 	__webpack_require__(225);
```

Comment:
Can you exclude browser bundles from this PR? It will be updated only at release.